app/assets/javascripts/select-all.js

```diff
@@ -2,17 +2,25 @@
   "use strict";
 
   Modules.SelectAll = function () {
-    var that = this;
-
-    that.start = function (element) {
+    this.start = function (element) {
       function addEventListener() {
         var selectAll = element.find('#select_all');
+        var selectAllPages = element.find('#select_all_pages');
```

Comment:
Could you please prefix jQuery variables with `$`, as demonstrated in our [documentation](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/docs/javascript.md#module-structure).